javascript/javascript1/week2/Student-manager.js

```diff
@@ -0,0 +1,49 @@
+
+    const class07Students = [];
+    function addStudentToClass(studentName) {
+        let str='';
+        let number=getNumberOfStudents();
+        if(studentName===''){
+            str='studentName can not be empty';
+        }else{
+            let flag=0;
+            for(let i=0;i<number;i++){
+                if(class07Students[i]===studentName){
+                    flag=1;
+                }
+            }
+            if(flag===1){
+                str='same person cannot be added';
+            }else{
+                if(studentName==='Queen in Denmark'){
+                    class07Students.push('Queen in Denmark');
+                    str='Queen in Denmark added succesfully';
+                }else{
+                    if(number<6){
+                        class07Students.push(studentName);
+                        str=studentName + ' added succesfully';
+                    }else{
+                        str='There can be only 6 students in a class';
+                    }
+                }
+            }
+        }
```

Comment:
To check if the student is already exists use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)